src/sentry/features/temporary.py

```diff
@@ -461,6 +461,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:trace-view-v1", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     # Enable feature to load new trace view in replay trace tab.
     manager.add("organizations:replay-trace-view-v1", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+    # Enable feature to use span only trace endpoint.
+    manager.add("organizations:trace-spans-only", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
let's call this `spans-format` since the other thing is being referred to as `useSpans`